packages/itwinui-css/src/button/base.scss

```diff
@@ -2,37 +2,28 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
+$disabledButtonSelectors: '[disabled], :disabled, [aria-disabled="true"], [data-iui-disabled="true"], .iui-disabled';
```

Comment:
I don't think we need `iui-disabled`. Is it used anywhere?

Reply:
@rohan-kadkol, do I recall correctly that you found somewhere `.iui-disabled` was being used by `.iui-button`?

Reply:
Our previous discussion was actually about `data-iui-disabled` and not `.iui-disabled` (https://github.com/iTwin/iTwinUI/pull/1919#discussion_r1548079112).

Searching for `.iui-disabled` in the react code, I found that it is used in `SplitButton`:

https://github.com/iTwin/iTwinUI/blob/e2e6ba7abbb46ac99ed225775b56942b88d8da04/packages/itwinui-react/src/core/Buttons/SplitButton.tsx#L114-L116

But I believe that can be removed from react since `.iui-disabled` is not in the disabled `SplitButton`'s HTML demo page.

Reply:
Removed from the button SCSS files (not split button).